src/common/pure/GnosisSafeLink/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { ExternalLink } from 'legacy/theme'
+
+import { getSafeWebUrl } from 'api/gnosisSafe'
+
+export function GnosisSafeLink(props: {
```

Comment:
It's no longer called `Gnosis Safe`, can you rename it?

I was thinking to name it `SafeLink`, but it's too ambiguous.
`SafeWalletLink` is more precise IMO

Reply:
Fair enough, renamed